bokehjs/src/coffee/source/geojson_data_source.coffee

```diff
@@ -0,0 +1,124 @@
+_ = require "underscore"
+{logger} = require "../common/logging"
+ColumnDataSource = require "./column_data_source"
+
+class GeoJSONDataSource extends ColumnDataSource.Model
+
+  initialize: (options) ->
+    @set('geojson', JSON.parse(options.geojson))
```

Comment:
geojson is a string on the python side, and will be kept in sync; this probably causes a change notification which tries to send the parsed JSON back to the server, I would expect that to generate warnings in a `bokeh serve` situation (in a standalone file situation it wouldn't). Also, if `geojson` is changed on the server later, this would be set back to a string, I think?

An alternative approach is the "computed properties" support, which is done with `@register_property` and `@add_dependencies`, for example here we set up a `min` property that depends on `start` and `end`: https://github.com/bokeh/bokeh/blob/caaa3c6941014bf25184ff300b632ea1b3ebe51b/bokehjs/src/coffee/range/range1d.coffee#L8-L11 so you'd make a property like `parsed_geojson` or something which is computed from the `geojson` string property.

Or maybe I'm confused entirely. :-)